cli/src/main/java/com/box/l10n/mojito/cli/command/checks/GlossaryTermCaseCheckerTrie.java

```diff
@@ -8,6 +8,8 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
+import static com.box.l10n.mojito.cli.command.extractioncheck.ExtractionCheckNotificationSender.QUOTE_MARKER;
```

Comment:
Wondering if we should call that "monospaced" (or inline code) in that case since it is the back tick. Trying to find the common ground 
- https://slack.com/help/articles/202288908-Format-your-messages
- https://secure.phabricator.com/book/phabricator/article/remarkup/

"Quote" or "Block code" is `>` in both, so `QUOTE_MARKER` would be kept it for that.

Wondering if in that case we could have even use the back-tick directly without substitution? I guess that does not work for  some other like bolding that don't share the exact same code hence doing that? Eye balling isn't that happening for the list?

Reply:
The individual notification service senders are applying the bolding in the message at present but the failure text from the checkers themselves has no bolding. 

We could use the backtick without substitution, if we do use a new notifier service in future that doesn't use the backtick that sender could do the replacement.